src/esp/geo/CoordinateFrame.cpp

```diff
@@ -4,16 +4,18 @@
 
 #include "CoordinateFrame.h"
 
+#include <utility>
+
 #include "esp/geo/geo.h"
 #include "esp/io/json.h"
 
 namespace esp {
 namespace geo {
 
-CoordinateFrame::CoordinateFrame(const vec3f& up /* = ESP_UP */,
-                                 const vec3f& front /* = ESP_FRONT */,
-                                 const vec3f& origin /* = vec3f(0, 0, 0) */)
-    : up_(up), front_(front), origin_(origin) {
+CoordinateFrame::CoordinateFrame(vec3f up /* = ESP_UP */,
+                                 vec3f front /* = ESP_FRONT */,
+                                 vec3f origin /* = vec3f(0, 0, 0) */)
+    : up_(std::move(up)), front_(std::move(front)), origin_(std::move(origin)) {
```

Comment:
Not sure about this change. The `vec3f` is (or should be, but it's Eigen so I can't tell) a trivially copyable type so `std::move()` is useless. And we're commonly passing vectors by `const&` everywhere so this causes inconsistency.

What's the clang-tidy warning that triggered this?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html